src/ui/Panel.tsx

```diff
@@ -664,6 +673,17 @@ export class Panel extends Object2D {
         this.eventEmitter.emit('axisPointerUpdate', this.activeAxisPointers);
     }
 
+    static showCoordinateError(visible: boolean, message : string = null) {
+        let element = document.querySelector<HTMLElement>('.valis-error-message'); 
+
+        if (!element) {
+            return;
+        }
+
+        element.style.display = visible ? 'inline' : 'none';
+        element.textContent = message ? message : element.textContent;
```

Comment:
Just one thought - feel free to disregard this - I realized that if you type in invalid coordinates, and get an error message, and then enter valid coordinates, the HTML markup preserves the error message for the invalid coordinates and just doesn't display it. Would it make sense to have an empty string as the error message for valid coordinates? I think you would set that here. On the other hand, maybe this doesn't matter at all because the user won't see error messages with "display: none".

Reply:
Done.